libcxx/utils/libcxx/test/features.py

```diff
@@ -894,6 +894,14 @@ def check_gdb(cfg):
             cfg.available_features,
         ),
     ),
+    # Tests that require __log_error support in the built library
+    Feature(
+        name="availability-log_error-missing",
+        when=lambda cfg: BooleanExpression.evaluate(
+            "!libcpp-has-no-availability-markup && (stdlib=apple-libc++ && !_target-has-llvm-21)",
+            cfg.available_features,
+        ),
+    ),
```

Comment:
Can you move this to the end of the feature list? They are roughly in chronological order right now and I think we should keep that.